databases/week2/main.sql

```diff
@@ -0,0 +1,62 @@
+-- Part 1: Working with tasks
+-- 1.Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+SET
+  NAMES utf8mb4;
+USE hyfdbweek2;
+ALTER TABLE
+  task
+ADD
+  user_id varchar(50);
+INSERT INTO
+  task (
+    title,
+    description,
+    created,
+    updated,
+    due_date,
+    status_id,
+    user_id
+  )
+VALUES
+  (
+    "Homework",
+    "Databases week2 homework",
+    "2022-03-13 06:54:16",
+    "2022-03-15 06:54:16",
+    "2022-03-18 06:54:16",
+    "1",
+    "rismita87"
+  );
+--2.Change the title of a task
+UPDATE
```

Comment:
Formatting suggestion, I wouldn't split the UPDATE statement from the TABLE NAME it updates. 

Pointer: improves code readability and clarity